lib/node_modules/@stdlib/math/base/special/lnf/test/test.js

```diff
@@ -0,0 +1,241 @@
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+	var e;
+
+	expected = mediumPositive.expected;
+	x = mediumPositive.x;
+	for ( i = 0; i < x.length; i++ ) {
+		y = lnf( x[i] );
+		e = float64ToFloat32( expected[ i ] );
+		if ( y === e ) {
+			t.equal( y, e, 'x: '+x[i]+', y: '+y+', expected: '+e );
+		} else {
+			delta = abs( y - e );
+			tol = EPS * abs( e );
+			t.equal( delta <= tol, true, 'within tolerance. x: '+x[i]+'. y: '+y+'. E: '+e+'. Δ: '+delta+'. Tolerance: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function evaluates the natural logarithm of `x` (small positive values)', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+	var e;
+
+	expected = smallPositive.expected;
+	x = smallPositive.x;
+	for ( i = 0; i < x.length; i++ ) {
+		y = lnf( x[i] );
+		e = float64ToFloat32( expected[ i ] );
+		if ( y === e ) {
+			t.equal( y, e, 'x: '+x[i]+', y: '+y+', expected: '+e );
+		} else {
+			delta = abs( y - e );
+			tol = 106.0 * EPS * abs( e );
+			t.equal( delta <= tol, true, 'within tolerance. x: '+x[i]+'. y: '+y+'. E: '+e+'. Δ: '+delta+'. Tolerance: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function evaluates the natural logarithm of `x` (smaller positive values)', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+	var e;
+
+	expected = smaller.expected;
+	x = smaller.x;
+	for ( i = 0; i < x.length; i++ ) {
+		y = lnf( x[i] );
+		e = float64ToFloat32( expected[ i ] );
+		if ( y === e ) {
+			t.equal( y, e, 'x: '+x[i]+', y: '+y+', expected: '+e );
+		} else {
+			delta = abs( y - e );
+			tol = 1.8 * EPS * abs( e );
+			t.equal( delta <= tol, true, 'within tolerance. x: '+x[i]+'. y: '+y+'. E: '+e+'. Δ: '+delta+'. Tolerance: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function evaluates the natural logarithm of `x` (tiny positive values)', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+	var e;
+
+	expected = tinyPositive.expected;
+	x = tinyPositive.x;
+	for ( i = 0; i < x.length; i++ ) {
+		y = lnf( x[i] );
+		e = float64ToFloat32( expected[ i ] );
+		if ( y === e ) {
+			t.equal( y, e, 'x: '+x[i]+', y: '+y+', expected: '+e );
+		} else {
+			delta = abs( y - e );
+			tol = EPS * abs( e );
+			t.equal( delta <= tol, true, 'within tolerance. x: '+x[i]+'. y: '+y+'. E: '+e+'. Δ: '+delta+'. Tolerance: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function evaluates the natural logarithm of `x` (subnormal values)', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+	var e;
+
+	expected = subnormal.expected;
+	x = subnormal.x;
+	for ( i = 0; i < x.length; i++ ) {
+		y = lnf( x[i] );
+		e = float64ToFloat32( expected[ i ] );
+		if ( y === e ) {
+			t.equal( y, e, 'x: '+x[i]+', y: '+y+', expected: '+e );
+		} else {
+			delta = abs( y - e );
+			tol = 75.0 * EPS * abs( e );
```

Comment:
This is a bit large. What is the delta of the double-precision variant for subnormals?